feat: Automate Artifactory/Nexus/Quay integration in ACS

Roming22 opened the pull request:
cf [RHTAPINST-93](https://issues.redhat.com//browse/RHTAPINST-93)

The `rhtap-integrations` chart now takes cares of cross products integrations. Resources that are needed to setup products have been moved to the `rhtap-infrastructure` chart.

The `rhtap-integrations` chart is now executed after all products have been deployed.

rh-pre-commit.version: 2.3.2
rh-pre-commit.check-secrets: ENABLED

rhopp commented:
I think this part of install.sh should be removed once this is merged https://github.com/redhat-appstudio/rhtap-cli/blob/main/integration-tests/scripts/install.sh#L123-L187
Also this PR deals only with `quay`  case, but it should also cover artifactory and nexus.

otaviof reviewed (approved):
/lgtm

otaviof commented:
/retest

Roming22 commented:
I'll handle @rhopp comment before merging the PR.

Roming22 commented:
/retest

otaviof commented on installer/charts/rhtap-integrations/values.yaml:
As a Helm best practice, you should keep the parent key (`integrations`) and rely on inner attributes.

Roming22 replied:
Fixed

otaviof reviewed (changes requested):
Just a small change regarding Helm best practices, that allow us to centralize all the Chart's attributes (values) on a single place.

otaviof reviewed (commented):
/lgtm

otaviof commented:
/retest

Roming22 committed:
feat: Automate Artifactory/Nexus/Quay integration in ACS

cf RHTAPINST-93

The `rhtap-integrations` chart now takes cares of cross products
integrations. Resources that are needed to setup products have been
moved to the `rhtap-infrastructure` chart.

The `rhtap-integrations` chart is now executed after all products have
been deployed.

rh-pre-commit.version: 2.3.2
rh-pre-commit.check-secrets: ENABLED

xinredhat commented:
[e2e-4.17-ffmzh](https://console.redhat.com/application-pipeline/workspaces/rhtap-shared-team/applications/rhtap-cli/pipelineruns/e2e-4.17-ffmzh) finished successfully. it's safe to merge it
